config/routes.rb

```diff
@@ -14,9 +12,10 @@
 get "user/login"
 get "user/sign_up"
 get "employee/employee"
-get "laptop/laptop"
 get "employees/index"
 
+get 'assets/laptops', to: 'assets#laptops', as: 'laptops'
```

Comment:
it picks up under  /assets/laptops in the views now, because the folder is still called assets in the views